libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -184,6 +184,21 @@
         LOGIN_INSERTED_INVALID_URL,
         LOGIN_AUTOFILL_CREDENTIALS_FILLED,
         LOGIN_AUTOFILL_CREDENTIALS_UPDATED,
+        LOGIN_PROLOGUE_VIEWED,
+        LOGIN_EMAIL_FORM_VIEWED,
+        LOGIN_EMAIL_RETRY_VIEWED,
+        LOGIN_MAGIC_LINK_OPEN_FORM_VIEWED,
```

Comment:
Is this referring to the "open email client" screen? If so, let's rename this to "LOGIN_MAGIC_LINK_OPEN_EMAIL_CLIENT_VIEWED" or "LOGIN_MAGIC_LINK_SENT_VIEWED" or similar. WDYT?